debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerQueryResultProcessor.java

```diff
@@ -268,6 +268,42 @@ private boolean hasNext(ResultSet resultSet) throws SQLException {
         return false;
     }
 
+    private Table dispatchSchemaChangeEventAndGetTableForNewMonitoredTable(TableId tableId) throws SQLException {
```

Comment:
Monitored -> Captured or Tracked. Let's use only use monitoring for actual (JMX) metrics related notions.